lib/services/app-scaffolding-extensions-service.ts

```diff
@@ -22,9 +24,22 @@ export class AppScaffoldingExtensionsService extends dependencyExtensionsService
 	public prepareAppScaffolding(): IFuture<void> {
 		return (() => {
 			var appScaffoldingConfig = this.$dependencyConfigService.getAppScaffoldingConfig().wait();
-			var afterPrepareAction = () => this.$childProcess.exec("npm install", {cwd: this.appScaffoldingPath });
+			var afterPrepareAction = () => this.afterPrepareAction(this.appScaffoldingPath);
 			this.prepareDependencyExtension(AppScaffoldingExtensionsService.APP_SCAFFOLDING_NAME, appScaffoldingConfig, afterPrepareAction).wait();
 		}).future<void>()();
 	}
+
+	public afterPrepareAction(cwdPath: string, directoryPath?: string):IFuture<void> {
+		return (() => {
+			var command = "npm install ";
+			if(directoryPath) {
+				command += directoryPath;
+			}
+			command += command + " --production";
```

Comment:
this way command will be duplicated 
either replace `+=` with `=` or use `command += " --production"`